sopel/plugins/handlers.py

```diff
@@ -269,11 +276,11 @@ def get_label(self):
         docstring, its first line is used as the plugin's label.
         """
         default_label = '%s plugin' % self.name
-        module_doc = getattr(self._module, '__doc__', None)
 
-        if not self.is_loaded() or not module_doc:
+        if not self.is_loaded() or not hasattr(self.module, '__doc__'):
             return default_label
 
+        module_doc = getattr(self.module, '__doc__', "")
```

Comment:
I think you're right

Reply:
Oh, no, actually the behavior you describe is preserved for `__doc__` defined as the empty string, but this does still have a bug for `__doc__ = None` because `cleandoc(None)` is an error. Collapsing the `None` case onto the empty string should result in the correct older behavior, but I'll look at it a few minutes longer and see if there's a less clever way to express that.

Reply:
Semantics should be the same as of 72fb9e5, and in 9353cb1 I added a test that fails when this is broken in that particular way.